parallel_cluster/README.md

```diff
@@ -31,9 +31,14 @@ $ ssm i-XXXXXXXXXXXXX
 ```
 
 ### Running the cluster
+> CLUSTER_NAME can only be alpha-numeric and must start with a letter
 
 ```shell
-$ ./bin/start-cluster.sh <CLUSTER_NAME>
```

Comment:
I just wanted to have devs/users instantiate a cluster fast without caring about those long flags (by using reasonable defaults, only requiring a cluster name) :_S

Reply:
Yes but how do they specify which config? And KeyName, which I can discuss further in the SSM comment below.

Reply:
IIRC Florian had separate config files for each cluster? (`tothill`, `dev`, etc...)

Reply:
I started off having different cluster configurations in the same `config` file, but that may not scale to the number of use cases we'll need to support.
In any case, I'd focus on one use case for now and split it up if/when we need to.

Reply:
Keyname now resolved through ssm (do we have a keyname that's a dud - as in no one has the private pair?)
Long flags no longer required.
At the moment only two different configs exist in the config file, I think that's pretty reasonable.